common/src/main/java/com/wynntils/handlers/chat/RecipientType.java

```diff
@@ -9,6 +9,7 @@
 public enum RecipientType {
     INFO(null, null, "Info"),
     CLIENTSIDE(null, null, "Clientside"),
+    NPC("(?:§r)?§7\\[\\d+\\/\\d+\\] §r§2.+: §r§..*", "(?:§r)?§8\\[\\d+\\/\\d+\\] .+: §r§..*", "NPC"),
```

Comment:
@kristofbolyai I just realized... how does this mix with chat tabs?

Reply:
I am not sure what the question is. All tab works fine, other tabs won't have this selected. But we need to add a GUI checkmark..

Reply:
The question was: "It is waaay point bedtime and I don't fully understand what I have written". :) But more seriously. Even though this is a separate RecipientType, it will not make sense to have a separate tab for this. There are just a handful of those in the game at all (and I think like half of them are in King's Recruit...) So in the chat tabs, these should be treated as INFO messages (which is where they would have ended up before this PR). 

My qustion is: do I need to make some changes in ChatTabs to get messages tagged as NPC recipient type to be treated as INFO messages? 

I don't think it makes any sense to exose this category to the user.

Reply:
Are not the NPC messages canceled? I am not sure what special handling info messages have. But we need to add this so people without the overlay can see npc dialogue in chat?

Reply:
Riiiight.. I'm not sure what I'm thinking. Let me check the code again. And try without the overlay.